route/route.go

```diff
@@ -969,3 +974,16 @@ func (r *Router) AddOTLPMuxxer(muxxer *mux.Router) {
 	otlpMuxxer.HandleFunc("/logs", r.postOTLPLogs).Name("otlp_logs")
 	otlpMuxxer.HandleFunc("/logs/", r.postOTLPLogs).Name("otlp_logs")
 }
+
+func getDatasetFromRequest(req *http.Request) (string, error) {
+	dataset := mux.Vars(req)["datasetName"]
+	if dataset == "" {
+		return "", fmt.Errorf("missing dataset name")
+	}
+	dataset, err := url.PathUnescape(dataset)
+	if err != nil {
+		return "", err
+	}
+	fmt.Println("dataset", dataset)
```

Comment:
Is this print line intended?

Reply:
Nope, it can be removed. 

'''suggestion
'''